examples/devices/lighty-actions-device/src/main/java/io/lighty/netconf/device/action/processors/ActionServiceDeviceProcessor.java

```diff
@@ -130,13 +132,13 @@ protected String convertNormalizedNodeToXmlString(final NormalizedNode normalize
         return getNetconfDeviceServices().getXmlNodeConverter().serializeRpc(actionOutput, normalizedNode).toString();
     }
 
-    protected Absolute getActionInput(final Absolute path, final ActionDefinition action) {
+    static Absolute getActionInput(final Absolute path, final ActionDefinition action) {
```

Comment:
Please move static method to above or below others protected methods in this class.

Reply:
No. Its possible to make `static` also some other methods and reorganize the class later.